### PR TITLE
[OKU-15] Allow admin to approve a review

### PR DESCRIPTION
**Pull request by enricodelarosa**

### CODE REVIEW

**Comment by idiglove** [comment]
I'll review this once Setup branch has been merged

File: controllers/reivewsController.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { RequestHandler } from "express";
+
+import { PrismaClient, Prisma } from "@prisma/client";
+const prisma = new PrismaClient();
+
+export const approveReview: RequestHandler = async (req, res) => {
+    const reviewId: number = Number(req.params.reviewId);
```
Comment (idiglove) [inline controllers/reivewsController.ts]: add type for req and res

File: controllers/reivewsController.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    // TODO? Turn not found checker to middleware
```
Comment (idiglove) [inline controllers/reivewsController.ts]: sorry what does this mean?

File: controllers/reivewsController.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            heading: `Review with id ${reviewId} has been approved.`,
+            data: approvedReview,
+        });
+    }
+
+    return res.status(500).send({
+        heading: "Server error",
+        message: "Please try again.",
+    });
+};
```
Comment (idiglove) [inline controllers/reivewsController.ts]: well written

**Review by idiglove — CHANGES_REQUESTED** [review]
some changes and questions

File: controllers/reivewsController.ts (hunk 2 above)
Comment (enricodelarosa) [reply controllers/reivewsController.ts]: Small not to myself sana that a not found middleware can be built for requests that require finding a row a table then proceeding with some logic. 

Is that okay to put it on the codebase or I should remove it and just make the note on my personal todo list?

Comment (idiglove) [reply controllers/reivewsController.ts]: Yea have it as //TODO, it's fine :)

**Review by idiglove — APPROVED** [review]
lgtm 🏄🏽

File: controllers/reivewsController.ts (hunk 1 above)
Comment (enricodelarosa) [reply controllers/reivewsController.ts]: Types added. I was using the Request and Response types without importing them from express so it threw an error. Importing them from express fixed it.

 When I used the Request and Response types without importing them from express, was I accessing a global type Request and Response? It threwan error saying that params was not a property declared on the Request Type. Importing them from express fixed it.

File: controllers/reivewsController.ts (hunk 2 above)
Comment (enricodelarosa) [reply controllers/reivewsController.ts]: Got it! Thank you

**Pull request merged**